batch_seqr_loader/utils.py

```diff
@@ -1,9 +1,54 @@
 """
 Utility functions for the seqr loader
 """
-
+import hashlib
 import os
+from os.path import join
+from typing import Iterable, Tuple
+import logging
 from google.cloud import storage
+import hailtop.batch as hb
+from hailtop.batch import Batch
+from hailtop.batch.job import Job
+
+logger = logging.getLogger(__file__)
+logging.basicConfig(format='%(levelname)s (%(name)s %(lineno)s): %(message)s')
+logger.setLevel(logging.INFO)
+
+
+GATK_VERSION = '4.2.0.0'
```

Comment:
Should we use 4.2.1.0, to take advantage of the better GCS integration for GenomicsDB?

> GenomicsDB update: GATK has now moved to the new version of GenomicsDB (version 1.4.1), with native cloud support (previously, it relied on the slower GCS Connector library). At this point, it should basically be indistinguishable from using a network file system.

Reply:
I haven't tested that yet! For the following PR :)